### PR TITLE
ANDAUTH-72 created layout for landscape mode for signin viewflipper

### PR DESCRIPTION
**Pull request by somenath1435**
Before:

Screenshot_20200309-181238

Now:

Screenshot_20200309-190249

### CODE REVIEW

**Comment by tmortagne** [comment]
The screenshot looks good indeed.

**Comment by InsanusMokrassar** [comment]
Hello, @somenath1435 . I have seen the changes and wished to do a few by myself. I will try hard to do my best to complete these changes as soon as possible

**Comment by InsanusMokrassar** [comment]
I have tried to refactor your improvement. It is almost the same (from a point of view of layouts), but it is having changed code related to oidc. First of all, I have changed the mechanism of showing OIDC button: now it is disabled in case of unavailability of OIDC auth
Screenshot_20200315-152911_XWiki_Authenticator
Screenshot_20200315-153050_XWiki_Authenticator
Screenshot_20200315-153011_XWiki_Authenticator
Screenshot_20200315-153015_XWiki_Authenticator

**Comment by InsanusMokrassar** [comment]
@somenath1435 are you agree to these changes?

**Comment by somenath1435** [comment]
@InsanusMokrassar looks great

**Comment by InsanusMokrassar** [comment]
> @InsanusMokrassar looks great

Ok, waiting for @tmortagne review

**Review by InsanusMokrassar — CHANGES_REQUESTED** [review]
I found the problem that after rotating of screen we are coming back to the server setting up screen... If it will be possible, please, fix it

**Review by tmortagne — APPROVED** [review]

**Comment by somenath1435** [comment]
@InsanusMokrassar I'll create a new jira ticket for this

**Comment by somenath1435** [comment]
@InsanusMokrassar I've alread made one jira ticket for the issue you mentioned above, otherwise in my opinion this PR is ready

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -144,6 +144,7 @@
     <!--viewflipper signin-->
     <string name="chooseLoginMethod">Choose the login method</string>
     <string name="continueWith">Or continue with</string>
+    <string name="continueWithLandsacpe">---- Or continue with ----</string>
```
Comment (InsanusMokrassar) [inline app/src/main/res/values/strings.xml]: Redundant line, we should remove it

File: app/src/main/java/org/xwiki/android/sync/utils/extensions/ViewGroup.kt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package org.xwiki.android.sync.utils.extensions
+
+import android.view.ViewGroup
+
+var ViewGroup.enabled: Boolean
```
Comment (InsanusMokrassar) [inline app/src/main/java/org/xwiki/android/sync/utils/extensions/ViewGroup.kt:5]: Maybe, we should rewrite it with `View` receiver

**Comment by InsanusMokrassar** [comment]
> @InsanusMokrassar I've alread made one jira ticket for the issue you mentioned above, otherwise in my opinion this PR is ready


You are right. I (or you) will made a small changes according to my last comments and merge it

**Pull request merged**